app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script.rb

```diff
@@ -14,7 +14,10 @@ def create_in_provider(manager_id, params)
       task_ids = EmsRefresh.queue_refresh_task(manager)
       task_ids.each { |tid| MiqTask.wait_for_taskid(tid) }
 
-      find_by!(:manager_id => manager.id, :manager_ref => job_template.id)
+      find_by!(:manager_id => manager.id, :manager_ref => job_template.id).tap do |jt|
+        playbook = manager.configuration_script_payloads.find_by(:name => params[:playbook])
+        jt.update_attributes(:parent => playbook) if playbook
```

Comment:
I agree. Is it under @durandom's radar? If this part is taken care of by the refresh, this PR can address the spec only.